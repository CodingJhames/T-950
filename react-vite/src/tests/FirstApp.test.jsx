const { render } = require("@testing-library/react");
const { FirstApp } = require("../FirstApp");

describe('Pruebas en FirstApp', () => { 


    test('Debe de hacer el match con el Snapshot', () => {

        const title = 'Hola, soy Chepito';
        const { getByText } = render( < FirstApp  title={title} /> );
        expect( getByText( title ).toBeTruthy ).toMatchSnapshot();

     })


    test('Debe mostrar el tÃ­tulo en un h1', () => { 

        const title = "Hola, soy Chepito";
        const {container, getByText } = render( < FirstApp  title={title} /> );
        expect( getByText( title ) ).toBeTruthy();

        const h1 = container.querySelector('h1');
        expect( h1.innerHTML).toContain( title );


     }) 
 })


























