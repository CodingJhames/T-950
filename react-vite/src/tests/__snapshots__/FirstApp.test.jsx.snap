// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en FirstApp Debe de hacer el match con el Snapshot 1`] = `undefined`;
